01-intro.md

```diff
@@ -1,14 +1,12 @@
 ---
-layout: lesson
-root: ../..
+layout: page
 title: Analyzing Patient Data
```

Comment:
@ashwinsrnth Could you take a look on https://github.com/swcarpentry/git-novice/pull/8? You need similar changes in this pull request or another one.